package.json

```diff
@@ -24,8 +24,10 @@
     "build": "tsc",
     "build:watch": "tsc --watch",
     "prepack": "npm run clean && npm run build",
-    "start": "node --experimental-json-modules dist/src/index.js",
+    "start": "node --experimental-json-modules dist/src/scripts/prestart.js",
     "dev": "concurrently nodemon 'tsc --watch'",
+    "migrations:generate": "prisma migrate dev",
+    "migrations:run": "prisma migrate deploy",
```

Comment:
These 2 command would fail without `DATABASE_URL` env

Reply:
Good point, fixed in f859f55.